src/main/java/fi/aalto/cs/apluscourses/ui/repl/ReplConfigurationForm.java

```diff
@@ -0,0 +1,137 @@
+package fi.aalto.cs.apluscourses.ui.repl;
+
+import static fi.aalto.cs.apluscourses.presentation.ReplConfigurationFormModel.showREPLConfigWindow;
+import static java.util.Objects.requireNonNull;
+
+import com.intellij.openapi.fileChooser.FileChooserDescriptor;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.ComboBox;
+import com.intellij.openapi.ui.TextFieldWithBrowseButton;
+import com.intellij.openapi.vfs.VirtualFile;
+import fi.aalto.cs.apluscourses.presentation.ReplConfigurationFormModel;
+import javax.swing.JCheckBox;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import org.jetbrains.annotations.NotNull;
+
+public class ReplConfigurationForm extends JPanel {
+
+  public static final String INFOLABEL_TEXT = "<html>"
+      + "This is an A+ Scala REPL configuration window. By default, the working "
+      + "directory and the dependencies, loaded to the REPL classpath belong to the Module, that "
+      + "it was started on. To change the behavior, use the checkbox at the bottom of the window."
+      + "</html>";
+
+  private ReplConfigurationFormModel model;
+
+  private TextFieldWithBrowseButton workingDirectoryField;
+  private ComboBox moduleComboBox;
```

Comment:
This causes some compiler warnings from other lines in this file. Changing it to `ComboBox<String>` fixes the issue I think.